README.rst

```diff
@@ -30,6 +30,7 @@ Quickstart
   for quick notes.
 + `gs` in visual mode pastes the current selection (character-wise, line-wise 
   or block-wise) into the scratch buffer.
++ to enable persistence set `g:scratch_persistence_file` to a valid file path, i.e. '/tmp/scratch.txt'
```

Comment:
Could we rephrase (and wrap to 80 characters) as:

> + By default the contents of the scratch window are lost when leaving Vim. To enable cross-session persistence, set the `g:scratch_persistence_file` option to a valid file path.

I feel we should also highlight this in the features section. Could we add the following to the features list above:

> + Optional persistence across sessions.